app/helpers/document_helper.rb

```diff
@@ -12,13 +12,4 @@ def filter_current(documents)
   def archive_reason_collection_for_radio_buttons
     Document.archive_reasons.keys.map { |k| [k, I18n.t(k)] }
   end
-
```

Comment:
Good spot! I was convinced we were still using that somewhere, but have checked and indeed we are not!